src/utils/server.ts

```diff
@@ -36,13 +37,16 @@ const makeServer = (): Express => {
   app.use("/api/v1/candidate", candidateRoutes);
 
   app.use("/api/v1/skills", skillRoutes);
-  app.use("/api/v1/auth", authRoutes);
   app.use("/api/v1/job", jobRoutes);
   app.use("/api/v1/company", companyRoutes);
   app.use("/api/v1/interview", interviewRoutes);
   app.use("/api/v1/application", applicationRoutes);
   app.use("/api/v1/employer", employerRoutes);
 
+  app.use("/api/v1/c/auth", cAuthRoutes);
+
+  app.use("/api/v1/e/auth", eAuthRoutes);
```

Comment:
I think if we make route changes, the mobile and webapp should be updated as well as similar PR's being opened.